### ISSUE TITLE
Pinning memory can cause errors for smaller GPUs

### ISSUE THREAD

**Issue by jorshi** [neuralaudio/hear-eval-kit#316]
I'm getting a memory error trying to run predictions: `RuntimeError: Caught RuntimeError in pin memory thread for device 0.`

Setting `pin_memory=False` in the `DataLoader` in `task_predictions.py` solves this issue. https://github.com/neuralaudio/hear-eval-kit/blob/174b9268f96d3b4de51cd3557f9063d64fe1bd02/heareval/predictions/task_predictions.py#L744

Is this just for speed? Can we make this optional?

**Comment by turian** [comment]
Yes, we can definitely make this option. We might consider making a config.py, separated into configs that affect results (this one doesn't) and ones that are basically just performance
